pkg/traces/config.go

```diff
@@ -434,6 +435,8 @@ func (c *InstanceConfig) otelConfig() (*config.Config, error) {
 	if len(c.Receivers) == 0 {
 		return nil, errors.New("must have at least one configured receiver")
 	}
+	c.Receivers[pushreceiver.TypeStr] = nil // add a hacky push receiver for when an integration
```

Comment:
I'd try to avoid to add the push_receiver always if possible. Maybe we can add a non-yaml field to the traces config that is set based on the integrations config?

Reply:
I wanted to do that initially, but cound't figure out a way that didn't require significant refac... Didn't occur to me to use a non-yaml field though. Thanks for the hint, will investigate

Reply:
- [ ] only insert push receiver for traces instances used by the frontend receiver integration

Reply:
took some doing, but got it done!